internal/gps/vcs_repo.go

```diff
@@ -80,27 +82,54 @@ func newVcsLocalErrorOr(msg string, err error, out string) error {
 }
 
 func (r *gitRepo) get(ctx context.Context) error {
-	out, err := runFromCwd(ctx, expensiveCmdTimeout, "git", "clone", "--recursive", "-v", "--progress", r.Remote(), r.LocalPath())
-	if err != nil {
-		return newVcsRemoteErrorOr("unable to get repository", err, string(out))
+	cmd := exec.CommandContext(
```

Comment:
relying on `CommandContext` represents a regression vs. our current logic, as it relies on `Process.Kill()` to terminate the subprocess. we used to do that, and it produced lots of corrupted repositories. what we have isn't still isn't without problems, but it's better than all that.